packages/EasyBankFiles/src/Parsers/DirectEntry/Parser.php

```diff
@@ -14,22 +15,42 @@ final class Parser extends AbstractLineByLineParser
     /**
      * @const Code for header line
      */
-    private const HEADER = 0;
+    private const HEADER = '0';
+
+    /**
+     * @const Minimal header line length (calculated from maximum substr arguments)
```

Comment:
`@const` doesn't actually exist and it doesn't specify the actual type of value the constant holds. We MUST use `@var int`.

Reply:
Done.